### ISSUE TITLE
Segmentation fault on Docker Alpine Arm64

### ISSUE THREAD

**Issue by OzySky** [DoctorEvidence/lmdb-js#122]
HI, Thanks for the great library!
When I tried using it on Alpine arm64, it installed OK.
But when I try to open a simple db it crashes :(

```
const lmdb = require('lmdb')
const db = lmdb.open('./test')

Segmentation fault (core dumped)
```

lmdbx seems to work, but last release was a month ago, and I don't know what the tradeoffs are.

**Comment by OzySky** [comment]
First few lines from LLDB core back trace (Tried both on Node 16 & 17 which makes abi93 a bit suspect) 
```
thread #1, name = 'node', stop reason = signal SIGSEGV
  * frame #0: 0x0000000000005b00
    frame #1: 0x0000ffffb1c7a430 node.abi93.node`mdb_env_open + 368
    frame #2: 0x0000ffffb1c5f7a4 node.abi93.node`EnvWrap::openEnv(int, int, char const*, char*, Compression*, int, int, unsigned long, int, char*) + 404
    frame #3: 0x0000ffffb1c5fd20 node.abi93.node`EnvWrap::open(Nan::FunctionCallbackInfo<v8::Value> const&) + 1168
    frame #4: 0x0000ffffb1c5a5c0 node.abi93.node`Nan::imp::FunctionCallbackWrapper(v8::FunctionCallbackInfo<v8::Value> const&) + 200
```

**Comment by OzySky** [comment]
running npm run recompile fixes the problem

**Comment by kriszyp** [comment]
I currently don't have any prebuilds for alpine arm64, as I am not sure how to compile for alpine arm64 on github actions (not sure if you have any ideas on how that can be done). And without prebuilds, it looks like it is attempting to use the glibc builds. node.abi93.node is glibc build for node 16 (node 17 should use node.abi102.node). And it would make sense that npm run recompile would fix this since it would provide musl build. I don't know why the lmdbx release would work, that seems like it should run into the same problem.

Ideally it would be nice to get alpine arm64 prebuilds working, but again, not sure how to do that. The next best solution would be to explicitly tag the glibc builds as such, so they don't get used on alpine, which should force compilation on install.

**Comment by OzySky** [comment]
It's possible to build on x64 alpine for arm64 using musl-cross.
I forked the repo and tested it. It seems to work great.
OzySky@c6fa51b
Of course adding the musl tag to the prebuildify process is needed.

**Comment by kriszyp** [comment]
This is fantastic, thank you for putting this together! I assume it is ok if I merge your commits (so you get credit for it).

**Comment by OzySky** [comment]
Great! Thanks again for this fantastic package.

**Comment by kriszyp** [comment]
This should be available in v2.1.3, let me know if you find any issues with it.
